eo.json

```diff
@@ -621,22 +621,22 @@
         "Importer.Model.Centimeters": "Centimetroj (cm)",
         "Importer.Model.Inches": "Coloj",
         "Importer.Model.FinalizePrompt": "Ĉu ĉio finita?",
-        "Importer.Model.RunImport": "Kuru Importi!",
+        "Importer.Model.RunImport": "Komenci Importadon!",
 
         "Importer.Model.Advanced.Scale": "Skalo:",
         "Importer.Model.Advanced.AutoScale": "Aŭtomata skalo",
         "Importer.Model.Advanced.MaxTexSize": "Maksimuma tekstura grandeco",
         "Importer.Model.Advanced.ImageFormat": "Bildformato:",
         "Importer.Model.Advanced.Material": "Materialo:",
-        "Importer.Model.Advanced.PreferSpecular" : "preferi Specular",
+        "Importer.Model.Advanced.PreferSpecular" : "Preferi Specular",
         "Importer.Model.Advanced.AlignAxis": "Vicigi Akson:",
 
         "Importer.Model.Advanced.Normals": "Kalkuli Normalaĵojn",
         "Importer.Model.Advanced.Tangents": "Kalkuli klavojn",
         "Importer.Model.Advanced.VertexColors": "Importi verticajn kolorojn",
         "Importer.Model.Advanced.Bones": "Importi Ostojn",
         "Importer.Model.Advanced.Lights" : "importi lumon",
-        "Importer.Model.Advanced.TextureAlpha": "Kalkuli Teksturan Alfa",
+        "Importer.Model.Advanced.TextureAlpha": "Kalkuli Teksturan Alfaon",
```

Comment:
PIV says [alfo](https://vortaro.net/#alfo_kd) is a plant, but has an entry for [alfa](https://vortaro.net/#alfa_kd), which I added the noun ending to because I thought preserving the "alfa" -> "alpha" correlation was important. Komputeko unfortunately doesn't have an entry for "alpha channel", so I'm not sure what would be best to do in this situation. Maybe "travideblecon"?

Reply:
This is an interesting topic.

I would suggest we use the translation name "alfo/alfao", instead of its descriptive name "travidebleco". Most of other languages translate it like that, so I could assume it is widely recognized by its original given name.

And concerning the Esperanto form we should use, I will state some points:

- Alfo is recognized as the [plant](https://reta-vortaro.de/revo/art/alf.html#alf.0o) and as the [greek letter](https://reta-vortaro.de/revo/art/alf1.html#alf1.0o) in ReVo. Sadly, it seems that PIV hasn't.

- The problem with the form shown in PIV "alfa" is that it is not an assimilated word in Esperanto. So it is actually a linguistic borrow from Greek, and not an actual Esperanto word. That's why it is more often used as a prefix, and doesn't appear as a root "alfa/".

- The form used in vikipedio is [alfokanalo](https://eo.wikipedia.org/wiki/Alfokanalo).

- And in PMEG the form used is "alf/o". This was my main reference.

However, I like the proposal of the root alfa/o, as you implied, and as suggested by Alano Gerardo in [ReVo](https://reta-vortaro.de/revo/art/alf1.html#alf1.0o). Plus, it would solve the problem of its plant homonym.

I don't see a problem from using one or the other. Time will tell which form will be used. But for now, you can choose whichever you prefer. I like your initial suggestion of "alfa/o", so maybe we stick with that.

Reply:
Thank you for the feedback! I'm surprised the entire Greek alphabet wasn't given Esperantized names. It would have certainly made things easier since a lot of STEM fields refer to it... Let's stick with alfao for now. Perhaps if Neos gets more Esperanto-speaking users, they can provide their input as well.